Source/Core/getAbsoluteURL.js

```diff
@@ -0,0 +1,41 @@
+/*global define*/
+define([
+        '../ThirdParty/Uri',
+        './defaultValue',
+        './defined',
+        './DeveloperError'
+    ], function(
+        Uri,
+        defaultValue,
+        defined,
+        DeveloperError) {
+    "use strict";
+
+    /**
+     * Given a URI, returns the extension of the URI.
+     * @exports getAbsoluteURL
+     *
+     * @param {String} url The Url.
+     * @param {String} base The base.
+     * @returns {String} The absolute url of the Url.
+     *
+     * @example
+     * //absolute url will be "http://localhost:8080/Specs/awesome.png";
+     * var absoluteURL = Cesium.getAbsoluteURL('awesome.png');
+     */
```

Comment:
This example doesn't seem correct. You should replace it with an example that uses both parameters.